frontend/src/routes/Home/Overview/OverviewPage.test.tsx

```diff
@@ -1115,8 +1115,8 @@ it('should render overview page with expected data', async () => {
 
   // Check Cluster violations chart rendered
   await waitFor(() => expect(getAllByText('Cluster violations')).toHaveLength(2))
-  await waitFor(() => expect(getByText('1 Without violations')).toBeTruthy())
-  await waitFor(() => expect(getByText('1 With violations')).toBeTruthy())
+  await waitFor(() => expect(getByText('1 No violations')).toBeTruthy())
```

Comment:
Can we change this particular case to be `with no violations` whenever there is a number before `No violations`?